edx_proctoring/static/index.js

```diff
@@ -9,19 +9,25 @@ export const handlerWrapper = (Handler) => {
       }
       case 'startExamAttempt': {
         if(handler.onStartExamAttempt) {
-          handler.onStartExamAttempt().then(() => self.postMessage({type: 'examAttemptStarted'}))
+          handler.onStartExamAttempt()
+                 .then(() => self.postMessage({type: 'examAttemptStarted'}))
+                 .catch(() => self.postMessage({type: 'examAttemptStartFailed'}))
```

Comment:
A very minor point, but do we have formatting guidelines that cover whether to use semicolons to terminate statements, and whether to use a space in `if (` ?  The files in this PR differ.

Reply:
Can we use https://github.com/edx/eslint-config-edx for this?

Reply:
We sure do; they're both implemented in eslint configs we don't have set up for this repo. Thanks for pointing these out.